awx/main/utils/filters.py

```diff
@@ -213,6 +210,8 @@ def query_from_string(cls, filter_string):
         filter_string_raw = filter_string
         filter_string = unicode(filter_string)
 
+        unicode_spaces = [unichr(c) for c in xrange(sys.maxunicode) if unichr(c).isspace()]
```

Comment:
There has _got_ to be a better way to do this :)

Reply:
Correct, see #838 now